### PR TITLE
Closes #423 - Improve error handling if user does not include all config...

### LINKED ISSUE #423: Improve error handling if user does not include all configuration files
I copied `fluo.properties`  to `conf/` directory but not every other config file which caused the following exception:

```
23:51:46,364 |-ERROR in ch.qos.logback.classic.joran.JoranConfigurator@58a8f7 - Could not open [fluo-1.0.0-beta-1-SNAPSHOT/conf/logback-stdout.xml]. java.io.FileNotFoundException: //fluo-1.0.0-beta-1-SNAPSHOT/conf/logback-stdout.xml (No such file or directory)
	at java.io.FileNotFoundException: /fluo-1.0.0-beta-1-SNAPSHOT/conf/logback-stdout.xml (No such file or directory)
	at 	at java.io.FileInputStream.open(Native Method)
	at 	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at 	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:75)
	at 	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:68)
	at 	at io.fluo.cluster.util.LogbackUtil.init(LogbackUtil.java:81)
	at 	at io.fluo.cluster.yarn.YarnAdmin.main(YarnAdmin.java:270)
```
I then fixed the configuration but received an exception when I ran start.
```
$ ./bin/fluo yarn start
Exception in thread "main" org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /yarn/app.id
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:111)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.zookeeper.ZooKeeper.getData(ZooKeeper.java:1155)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:302)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:291)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.GetDataBuilderImpl.pathInForeground(GetDataBuilderImpl.java:287)
	at org.apache.curator.framework.imps.GetDataBuilderImpl.forPath(GetDataBuilderImpl.java:279)
	at org.apache.curator.framework.imps.GetDataBuilderImpl.forPath(GetDataBuilderImpl.java:41)
	at io.fluo.cluster.yarn.YarnAdmin.getAppId(YarnAdmin.java:257)
	at io.fluo.cluster.yarn.YarnAdmin.getAppInfo(YarnAdmin.java:236)
	at io.fluo.cluster.yarn.YarnAdmin.logExistsButNotRunning(YarnAdmin.java:232)
	at io.fluo.cluster.yarn.YarnAdmin.start(YarnAdmin.java:75)
	at io.fluo.cluster.yarn.YarnAdmin.main(YarnAdmin.java:297)
```

### PR DESCRIPTION
**Pull request by mikewalch**
...uration files

### CODE REVIEW

File: modules/cluster/src/main/java/io/fluo/cluster/mini/MiniAdmin.java [hunk 1]
```diff
@@ -42,6 +43,8 @@ public static void main(String[] args) {
         System.exit(-1);
       }
       options.validateConfig();
+      
+      ClusterUtil.verifyConfigFilesExist(options.getConfigDir(), "fluo.properties", "log4j-stdout.xml");
```
Comment (keith-turner) [inline modules/cluster/src/main/java/io/fluo/cluster/mini/MiniAdmin.java]: For the log config files, could verify their existance in `Log4jUtil.init()` and `LogbackUtil.ini()`.  Then the code calling those methods would not have to make assumptions.

Comment (mikewalch) [inline modules/cluster/src/main/java/io/fluo/cluster/mini/MiniAdmin.java]: Good idea.. I will add a fix for this.

Comment (mikewalch) [inline modules/cluster/src/main/java/io/fluo/cluster/mini/MiniAdmin.java]: Fixed in 38821d8c2d747

**Comment by keith-turner** [comment]
+1

**Pull request merged**